PaTTAN Room Displays/PaTTAN Room Displays/RoomDataService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using System.Xml.Linq;
+
+namespace PaTTAN_Room_Displays
+{
+    internal class RoomDataService
+    {
+        public List<Meeting> GetRoomData()
```

Comment:
This is harmless, but it's odd to have a **public** constructor for an **internal** class

Reply:
I'm really thinking this should be something that is injected rather than instantiated.